autodocs/templates/image_provenance_page.tpl

```diff
@@ -0,0 +1,65 @@
+---
+title: "Provenance Information for {{ title }} Images"
+type: "article"
+description: "{{ description }}"
+date: 2022-11-01T11:07:52+02:00
+lastmod: 2022-11-01T11:07:52+02:00
+draft: false
+tags: ["Reference", "Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "{{ title }}"
+weight: 600
+toc: true
+---
+
+All Chainguard Images contain verifiable signatures and high-quality SBOMs (software bill of materials), features that enable users to confirm the origin of each image built and have a detailed list of everything that is packed within.
+
+## Verifying Image Signatures
+The **{{ title }}** Chainguard Images are signed using Sigstore, and you can check the included signatures using `cosign`.
+
+The following command requires [cosign](https://docs.sigstore.dev/cosign/overview/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine. It will pull detailed information about all signatures found for the provided image.
+
+```shell
+cosign verify --certificate-oidc-issuer=https://token.actions.githubusercontent.com --certificate-identity=https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main cgr.dev/chainguard/{{ title }} | jq
+```
+
+By default, this command will fetch signatures for the `latest` tag. You can also specify the tag you want to fetch signatures for.
+
+## Downloading and Verifying SBOMs
+
+All Chainguard Images come with a high-quality Software Bill Of Materials (SBOM) generated at build-time. The SBOM can be downloaded using the cosign tool:
+
+```shell
+cosign download sbom --platform linux/amd64 cgr.dev/chainguard/{{ title }} | jq
```

Comment:
Should this `--platform linux/amd64` be there? We have linux/arm64 images too right?

Reply:
Yeah this was from the first time I build these provenance pages, and we only had amd64 at that time... I think it's safe to remove it.